packages/tools/src/types/AnnotationTypes.ts

```diff
@@ -28,41 +28,11 @@ type Annotation = {
   /** If the annotation is auto generated from other annotations*/
   autoGenerated?: boolean;
   /** Metadata for annotation */
-  metadata: {
-    /**
-     * The position of the camera in world space
-     */
-    cameraPosition?: Types.Point3;
-    /**
-     * The focal point of the camera in world space
-     */
-    cameraFocalPoint?: Types.Point3;
-    /**
-     * The normal on which the tool was drawn
-     */
-    viewPlaneNormal?: Types.Point3;
-    /**
-     * The viewUp on which the tool was drawn.
-     */
-    viewUp?: Types.Point3;
-    /**
-     * The FrameOfReferenceUID
-     */
-    FrameOfReferenceUID: string;
+  metadata: Types.ViewTarget & {
```

Comment:
Should the viewTarget be a child?  I'd like to do so, but there are SO many places it is set.